### PR TITLE
[ES] Add `C` missing stubs

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing C stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish

SKIP_WIKILINK_CHECK

### CODE REVIEW

File: wiki/Community/Forum/Modding_Queues/es.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+stub: true
+---
+
+# Colas de modding
+
+[**Modding Queues**](https://osu.ppy.sh/community/forums/60) es un foro dedicado a solicitudes para la [revisión de un beatmap](/wiki/Modding) o mods. Consiste en hilos del foro o *colas* que pertenecen a diferentes modders y mappers. Cada cola tiene requisitos y límites individuales para obtener mods, que los visitantes deben respetar para poder recibir un mod.
```
Comment (agatemosu) [inline wiki/Community/Forum/Modding_Queues/es.md:7]:
```suggestion
[**Modding Queues**](https://osu.ppy.sh/community/forums/60) es un foro dedicado a solicitudes de [revisiones de beatmaps](/wiki/Modding) o mods. Consiste en hilos del foro o *colas* que pertenecen a diferentes modders y mappers. Cada cola tiene requisitos y límites individuales para obtener mods, que los visitantes deben respetar para poder recibir un mod.
```

File: wiki/Client/Beatmap_editor/Recalculate_slider_lengths/es.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Recalcular las longitudes de los sliders
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Recalculate_slider_lengths/es.md:5]:
```suggestion
# Recalcular la longitud de los sliders
```

File: wiki/Client/Beatmap_editor/Resnap/es.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Reajustar
+
+**Reajustar** se refiere a un par de comandos en la pestaña `Timing` en el [editor de beatmaps](/wiki/Client/Beatmap_editor). Uno de los cuales se aplica a todo el [beatmap](/wiki/Beatmap) y el otro a la sección de temporización actual. Si el beatmap tiene solo una sección de temporización, ambos comandos son iguales.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Resnap/es.md:7]:
```suggestion
**Reajustar** hace referencia a un par de comandos en la pestaña `Timing` en el [editor de beatmaps](/wiki/Client/Beatmap_editor). Uno de los cuales se aplica a todo el [beatmap](/wiki/Beatmap) y el otro a la sección de tiempo actual. Si el beatmap tiene solo una sección de tiempo, ambos comandos son iguales.
```

File: wiki/Community/Community_Mentorship_Program/es.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+tags:
+  - mentoring
+  - mentoría
+---
+
+# Community Mentorship Program
+
+El **Community Mentorship Program** es una forma para que los mappers sin experiencia interactúen con la comunidad de mapping a través de la mentoría de mappers o modders experimentados.
+
+El programa se organiza a través de Discord y se gestiona en ciclos estacionales. Cada ciclo se anuncia a través de una publicación de foro y los ciclos ocasionales se anuncian a través de publicaciones de noticias en la portada.
```
Comment (agatemosu) [inline wiki/Community/Community_Mentorship_Program/es.md:12]:
```suggestion
El programa se organiza a través de Discord y se gestiona en ciclos estacionales. Cada ciclo se anuncia a través de una publicación en el foro y los ciclos ocasionales se anuncian a través de publicaciones de noticias en la portada.
```

**Comment by agatemosu** [comment]
that and can be merged

**Review by agatemosu — APPROVED** [review]
CI fixed in https://github.com/ppy/osu-wiki/pull/10394

**Review by TicClick — APPROVED** [review]

**Pull request merged**